src/main/resources/assets/botania/lang/en_US.lang

```diff
@@ -1433,6 +1434,7 @@ item.botania:slimeBottle.name=Slime in a Bottle
 item.botania:starSword.name=Starcaller
 item.botania:exchangeRod.name=Rod of the Shifting Crust
 item.botania:magnetRingGreater.name=Greater Ring of Magnetization
+item.botania:shadowRod.name=Rod of the Corrupted Shadow
```

Comment:
Names of rods must resemble some sort of possible geological location or event, this name doesn't fit.